content/posts/07-meet-mkmr.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "Introducing mkmr, a collection of python3 tools to interact with GitLab"
+date: 2020-08-12T18:44:02-03:00
+draft: false
+---
+
+The Linux distribution I contribute to, [Alpine Linux](https://alpinelinux.org),
+has recently switched fully to using GitLab (self-hosted instance) and has started
+gradually phasing out support for sending patches via the Mailing List.
+
+To help smooth the transition for Mailing List contributors and also make contributing
+to Alpine Linux in general more efficient, I decided to step and help by making it easier
+for making Merge Requests, editing various attributes of the merge request and merging
```

Comment:
```suggestion
to create Merge Requests, editing various attributes of the merge request and merging
```

The capitalization on Merge Request and Mailing List seems a bit inconsistent, maybe you could either always write it capitalized it or write it in lower case?